File: src/main/java/de/scandio/e4/virtualusers/VirtualUserA.java
package de.scandio.e4.virtualusers;

import de.scandio.e4.interfaces.Scenario;
import de.scandio.e4.interfaces.VirtualUser;

import java.util.List;

public class VirtualUserA implements VirtualUser {

    @Override
    public List<Class<Scenario>> getScenarios() {

        return null;
    }
}

File: src/main/kotlin/de/scandio/e4/virtualusers/VirtualUserA.kt
package de.scandio.e4.virtualusers

import de.scandio.e4.worker.interfaces.Scenario
import de.scandio.e4.worker.interfaces.VirtualUser

class VirtualUserA : VirtualUser {
    override fun getScenarios(): List<Scenario> {
        val list = arrayListOf<Scenario>()
        return list
    }
}